tests/data/wire/ext_conf.xml

```diff
@@ -3,19 +3,19 @@
     <GAFamily>
       <Name>Prod</Name>
       <Uris>
-          <Uri>http://manifest_of_ga.xml</Uri>
+          <Uri>http://mock-goal-state/manifest_of_ga.xml</Uri>
```

Comment:
see mock_wire_protocol.py for the motivation of these updates in the test data

Reply:
Thanks for creating that file, really helpful!